UpdateDependencies.ps1

```diff
@@ -16,7 +16,7 @@ param(
     [string]$GitHubProject='corefx',
     [string]$GitHubUpstreamBranch='master',
     # a semi-colon delimited list of GitHub users to notify on the PR
-    [string]$GitHubPullRequestNotifications='')
+    [string]$GitHubPullRequestNotifications='stephentoub;ericstj;ellismg;weshaggard;joshfree')
```

Comment:
FYI - it is updated.  https://github.com/eerhardt/versions/blob/master/Subscriptions.json#L29